ios/platform/FBSDKCoreKit.framework/Headers/FBSDKAccessToken.h

```diff
@@ -101,6 +107,11 @@ FBSDK_EXTERN NSString *const FBSDKAccessTokenChangeNewKey;
  */
 @property (readonly, copy, nonatomic) NSString *userID;
 
+/**
+ Returns whether the access token is expired by checking its expirationDate property
+ */
+@property (readonly, assign, nonatomic, getter = isExpired) BOOL expired;
```

Comment:
We should expose this one as well, via `isExpired()`

Reply:
So would this `isExpired()` also be considered and defined as a method, or would it be treated the same as the other property getters such as userID or token?

Reply:
In this case, it would be a method only. We should talk about how properties and methods are generally exposed via the native API's :).